### ISSUE TITLE
Menu Not Working on Mobile

### ISSUE THREAD

**Issue by jclaybou** [wpexplorer/earth-wordpress-theme#3]
My menu works fine on a desktop browser, but in mobile devices it completely disappears. Check it out at joshclaybourn.com. What can I do to fix it? Thanks!

**Comment by vskaarthi** [comment]
You started seeing this issue after upgrading to wordpress 5.5? Just curious and may have a suggestion if that was the case.

**Comment by jclaybou** [comment]
I believe so.

**Comment by vskaarthi** [comment]
We ran into different issue with Wordpress 5.5 Upgrade. We installed plugin named "Enable jQuery Migrate Helper" for time being to have the site functional. Its an official plugin from Wordpress for clients like us to take some time to fix our deprecated functions in our Theme and Plugins.

I found this useful, see if this will resolve the issue for time being. While Aj/Team may find time to provide a permanent solution for the theme.

Link to the plugin - https://wordpress.org/plugins/enable-jquery-migrate-helper/
